### ISSUE TITLE
Module 'Fullwidth Slider' sliders un-editable

### ISSUE THREAD

**Issue by macmormedia** [elegantthemes/Divi-Beta#245]
**### Problem:**

Unable to edit individual sliders in the full width slider module. Admittedly this module wasn't created using the VB. There are also associated anomalies with the visual representation of this module (see screen shot and notes below).

**### Steps To Reproduce:**
1. Edit Front page of website in admin area and clicked on Use Visual Builder.
2. Clicked the Module Settings icon for the Module 'Fullwidth Slider'.
3. Clicked on the Slider Settings icon. No updated screen showed, however, the title changed to 'Slider Settings'.
4. Clicked the return arrow on the top-left and tried again. This time when clicking on the options icon it doesn't open anything, no title change at all.

**### Additional Information:**

Device: Desktop PC

Operating System: Win 10

Browser: Chrome Stable Release (current updates applied)

Screen Resolution: 1920 x 1080

**Link To Download Layout JSON File:** 

JSON file: https://drive.google.com/open?id=0BxvYSxcVUCjMRGU0LXJ4Xy1vbkk
Console Log: https://drive.google.com/open?id=0BxvYSxcVUCjMYzBKNlktb3FWWFE
CPU Profile: https://drive.google.com/open?id=0BxvYSxcVUCjMNjZ5XzdtOGRGb1k

**Screenshots:**

Before loading VB:

.PNG https://drive.google.com/open?id=0BxvYSxcVUCjMRjN6VG85TndyemM

After loading VB:

.PNG https://drive.google.com/open?id=0BxvYSxcVUCjMNDRZVWtLb2YyX0E

**Other Notes:**

- I have Divi Booster and KK Extended Modules currently active in this site. If necessary I will disable these and the other plugins that are activated. E.g.  Divi 100 Plugins.

- The screen has a full width page which gets 'squished' after loading VB.

- The directional arrows for the slider disappear after activating VB.

- Menu module text is pulled across to the section above the slider.

**Comment by timayhelen** [comment]
@macmormedia This isn't something I am able to replicate. Can you try temporarily disabling all of your plugins at once to see if it still occurs?

**Comment by macmormedia** [comment]
@timayhelen Yes - this worked - I had to clear browser cache and purge the server memcache as well for it work. I found by slowly adding the plugins back in one-by-one that a few plugins were causing loading issues:

- Animate It! plugin was influencing the rendering of text modules while VB was enabled

- The 'Divi Builder' plugin that was causing various conflicts (I still don't know why I had this activated).

- KK Divi Mods (as you probably already know) was causing the page width to become restricted among other disruptions to the rendering of modal windows. This will only be a problem if and when I need to use this plugin again for more client websites.

- On the plus side of 3rd party plugins, Divi Booster still works without any noticeable conflict.
